scout/constants/variant_tags.py

```diff
@@ -49,6 +49,79 @@
 
 VARIANT_CALL = ("Pass", "Filtered", "Not Found", "Not Used")
 
+# Formatting for different filters in the vcf file
+VARIANT_FILTERS = {
+    "pass": {"label": "PASS", "description": "Passed filtering", "label_class": "success"},
+    "germline": {"label": "GERM", "description": "Germline variant", "label_class": "warning"},
+    "germline_risk": {"label": "GERM", "description": "Germline risk", "label_class": "warning"},
+    "fail_nvaf": {
+        "label": "N",
+        "description": "Too high VAF in normal sample",
+        "label_class": "danger",
+    },
```

Comment:
We leave this filter to be applied by Clinicians. Generally speaking, if a sample has high normal_vaf then Scout's filter will be used to filter them out.